api/cloud/oc_cloud_rd.c

```diff
@@ -208,60 +208,60 @@ publish_published_resources(void *data)
   return OC_EVENT_CONTINUE;
 }
 
+#ifdef OC_DEBUG
 static void
 delete_resources_handler(oc_client_response_t *data)
 {
-  oc_cloud_context_t *ctx = (oc_cloud_context_t *)data->user_data;
   OC_DBG("[CRD] delete resources handler(%d)\n", data->code);
-
-  if (!(ctx->store.status & OC_CLOUD_LOGGED_IN))
-    return;
-  if (data->code != OC_STATUS_DELETED)
-    goto error;
-  rd_link_free(&ctx->rd_delete_resources);
-error : {
-}
 }
+#endif /* OC_DEBUG */
 
 static void
-delete_resources(oc_cloud_context_t *ctx, bool all)
```

Comment:
All need to be there - it means all resources at cloud must be unpublished
If all is not set then only selected resources need to be deleted.

Reply:
All was never used.

Reply:
I know but API counts with it. Maybe put there comment about that device can unpublish all resources without ins

Reply:
API? But it's a static helper function, surely that cannot be regarded as an API function.